scripts/verify-payouts.py

```diff
@@ -0,0 +1,100 @@
+
+# ========================================= MAIN EXECUTION =========================================
+
+# Get expected total match amount
+expected_total_dai_amount = Decimal(0)
+for (index,payout) in enumerate(expected_payouts):
+  # Skip the first one since we intentionally duplicated it in our payment mapping to ensure it gets
+  # filtered out when parsing event logs later
+  if index != 0:
```

Comment:
Extreme nitpick (no need to change) but I would have done this the opposite way:

```python
if index == 0:
  continue

expected_total_dai_amount += Decimal(payout['amount'])
```

I find it's easier to read/reason about code if the "happy path" is left indented.

Reply:
I actually did it that way originally, then decided this was simple enough that I'd rather save lines 😁

But this will be removed anyway when we port it over to gitcoinco/web

Reply:
Turns out it also got removed in 0c16c49ff0582ca6ba85537a4a489cbd265d8401!